source/jormungandr/tests/journey_common_tests.py

```diff
@@ -1208,3 +1207,50 @@ def test_strange_line_name(self):
         self.is_valid_journey_response(response, query)
 
         assert len(response['journeys']) == 1
+
+@dataset({"main_routing_test": {}})
+class JourneyMiniBikeMiniCar(object):
+    def test_activate_deactivate_min_bike_mini_car(self):
+        """
+        journeys from B to D
+        """
+        b_coord = "0.0002694935945864127;8.98311981954709e-05"
+        d_coord = "0.0002694935945864127;0.0"
+        sub_query = 'journeys?from={from_coord}&to={to_coord}&datetime={datetime}'.format(from_coord=b_coord,
+                                                                                          to_coord=d_coord,
+                                                                                          datetime="20120614T080000")
+        # Deactivate mini_bike and mini_car
+        query = '{}&last_section_mode[]=bike&first_section_mode[]=bike'.format(sub_query)
```

Comment:
must also be tested with
  * `'{}&last_section_mode[]=bike`
  * `'{}&first_section_mode[]=bike'`
  * `'{}&last_section_mode[]=walking&first_section_mode[]=bike'`
  * `'{}&last_section_mode[]=bike&first_section_mode[]=walking'`

Reply:
ok